ctk/common/src/main/kotlin/org/codice/compliance/utils/TestCommon.kt

```diff
@@ -151,8 +169,22 @@ class TestCommon {
             return URI(currentSPIssuer).host
         }
 
-        fun acsUrl(binding: SamlProtocol.Binding): String? {
-            return currentSPEntityInfo.getAssertionConsumerService(binding)?.url
+        /**
+         * Returns the Assertion Consumer Service URL or the Logout Service URL
+         *
+         * @param response - the response dom, if provided, used to determine if it's a login
+         * Response (ACS URL) or a LogoutResponse (Logout Service URL)
```

Comment:
Forgot about this comment and meant to come back to it to explain what alternative I'd recommend... Could we just keep the original `getACSUrl` method and create a new one for Logout instead of having the extra business logic here?

Reply:
I kind of thought the same thing. From our perspective (we implemented SLO after SSO), it makes sense to have the default behavior be SSO. However, from an outsider they won't know that.

I am between two solutions: making every class that calls this method pass in their response, or adding a comment explaining that SSO is the default. I am leaning towards the latter, because it's easier.

Reply:
I opted to have one method because the solution I had before required doing the "is it SSO or SLO login" every time this method is called. I can just have everyone that calls this method pass in a binding and response dom.

Reply:
Looked into it further. I think I'd prefer to stick with the one method but require they pass in the `responseDom`. The only places we're not passing it right now are below and I put what I'd suggest we do about it.

`SingleSignOnProfileVerifier.verifyBearerSubjectConfirmations` - Change it to pass the `responseDom` in.
`TestCommon.createDefaultAuthnRequest` - Have it not use this method. Just have it call `entityInfo.getAssertionConsumerService(binding)?.url` directly.